### PR TITLE
avoid mutating given payload

### PR DESCRIPTION
**Pull request by ericls**
#522

### CODE REVIEW

File: test/notification/index.js [hunk 1]
```diff
@@ -77,6 +77,17 @@ describe("Notification", function() {
 			});
 		});
 
+    describe("when aps properties are given by setters", function() {
```
Comment (oliversalzburg) [inline test/notification/index.js]: It seems like the indentation of this code block does not match the indentation of the surrounding code.

**Comment by ericls** [comment]
Fixed indentation error.

**Comment by ericls** [comment]
Fixed quote style

**Comment by oliversalzburg** [comment]
LGTM

**Pull request merged**